optimize/self-managed/optimize-deployment/migration-update/instructions.md

```diff
@@ -4,16 +4,16 @@ title: "Instructions"
 description: "Find out how to update to a new version of Optimize without losing your reports and dashboards."
 ---
 
-Roughly every quarter of a year a new minor version of Optimize is released. These documents guide you through the process of migrating your Optimize from one Optimize minor version to the other.
+Optimize releases two new minor versions a year. These documents guide you through the process of migrating your Optimize from one Optimize minor version to the other.
```

Comment:
This is probably a change we want to backport.

Reply:
Do you think we should backport this info to all versions?

Reply:
@misiekhardcore 😬 Do you know when Optimize changed the release cadence? 3.7+?

Reply:
Add the backport in the latest commit.